schema/config-windows.json

```diff
@@ -53,19 +53,31 @@
                                 "$ref": "defs.json#/definitions/uint64"
                             }
                         }
+                    }
+                }
+            },
+            "network": {
+                "id": "https://opencontainers.org/schema/bundle/windows/network",
+                "type": "object",
+                "properties": {
+                    "endpointList": {
+                        "id": "https://opencontainers.org/schema/bundle/windows/network/endpointList",
+                        "$ref": "defs.json#/definitions/ArrayOfStrings"
                     },
-                    "network": {
-                        "id": "https://opencontainers.org/schema/bundle/windows/resources/network",
-                        "type": "object",
-                        "properties": {
-                            "egressBandwidth": {
-                                "id": "https://opencontainers.org/schema/bundle/windows/resources/network/egressBandwidth",
-                                "$ref": "defs.json#/definitions/uint64"
-                            }
-                        }
+                    "allowUnqualifiedDNSQuery": {
+                        "id": "https://opencontainers.org/schema/bundle/windows/network/allowUnqualifiedDNSQuery",
+                        "$ref": "defs.json#/definitions/bool"
```

Comment:
[Travis doesn't like `bool`][1].  I think you want `"type": "boolean"`.

[1]: https://travis-ci.org/opencontainers/runtime-spec/jobs/230971925#L400